.travis.yml

```diff
@@ -1,67 +1,71 @@
-git blanguage: python
-sudo: required
-# if the https://travis-ci.org/ODM2/ODM2PythonAPI/requests ever says: missing config
```

Comment:
We don't really need `sudo` and without it Travis-CI can run in a faster using the container infrastructure. 

PS: I removed all the unnecessary comments.